src/components/MessageList/MessageList.tsx

```diff
@@ -718,22 +820,29 @@ const MessageListWithContext = <
             )}
           </View>
         }
-        ListFooterComponent={FooterComponent}
+        ListFooterComponent={() =>
```

Comment:
onEndReachedInProgress I think it makes more sense to have a default FooterComponent then only hsouw it when onEndReachedInProgress? Or some logic that let's a user basically replace ours but still have access to the logic, e.g. `() => <FotterComponent onEndReachedInprogress />`

Reply:
Hmm good point indeed. I will go with the first approach  (`to have a default FooterComponent`).